E3Next/Processors/Basics.cs

```diff
@@ -862,6 +862,8 @@ public static void CheckManaResources()
                 {
                     return;
                 }
+                var manastoneExceptionZones = new HashSet<string> { "poknowledge", "thevoida" };
```

Comment:
needs to be moved somewhere else and made static, otherwise your creating a ton of garbage, as this method is called a lot

Reply:
also might as well add it the character settings (i wish to avoid the gen settings for the most part)